sdk/formrecognizer/ai-form-recognizer/tests.yml

```diff
@@ -17,9 +17,3 @@ jobs:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        ENDPOINT: $(ENDPOINT)
```

Comment:
@jeremymeng are we set up for live testing samples yet? I think this change here will enable them.

Reply:
You are right! Our samples are not ready yet for CI.  Need to add `TestSamples: false`

Reply:
There are 3 options here

* Disable samples in form recognizer for now 
* Move all samples up to Node 10 where there is async iterator support 
* Override the whole live test matrix in the tests.yml file here so we can run samples in Node 10. This is the worst of the options because we're trying to wrangle the matrices into a consistent place.

Reply:
Ha... Now I reread it I can see we just need a `TestSamples: false` .. Plz ignore the other comment though it may be applicable to other services' tests.

Reply:
Further discussion has revealed that there are other samples that use async iterators and we should move samples testing up to Node 10. 

I've opened up https://github.com/Azure/azure-sdk-for-js/issues/8635 to track coming to a decision on whether we should do something different here.